templates/etc/apache2/sites-available/default.conf.j2

```diff
@@ -12,7 +12,8 @@
 
 {% if item.redirect_http|d() %}
     {{ apache__tpl_macros.get_redirect( item.redirect_http_code|d(307), "/", item.redirect_http) | indent(4) }}
-{% elif item.redirect_to_https|d(True) | bool %}
+{% elif item.redirect_to_https|default(apache__redirect_to_https) | bool %}
```

Comment:
I can. I think it's not good though. I, for one, needed to find out what `d` stands for. And as the only benefit of that shorthand, I see six saved characters.  The cost of having to type and read extra six characters will have amortised as soon as only a single person will not have to look up what `d` means.
But of course, my voice is not worth much because I'm not the one who has to maintain that ;-)